invenio_cern_sync/groups/sync.py

```diff
@@ -66,6 +67,7 @@ def sync(**kwargs):
         log_uuid=log_uuid,
     )
     roles_ids = create_or_update_roles(_serialize_groups(groups))
+    db.session.commit()
```

Comment:
Minor: Not sure why we need this one here?

´create_or_update_roles´ is alreay commiting at the end unless I am missing something: see https://github.com/inveniosoftware/invenio-oauthclient/blob/05b8b581d6b3fd1b74e53c01db376086d1f8c510/invenio_oauthclient/handlers/utils.py#L155

Reply:
You are right, removing it